USAGE.md

```diff
@@ -484,6 +484,133 @@ $ tw  participants update --name=collaborator@mydomain.com --type=COLLABORATOR -
   Participant 'collaborator@mydomain.com' has now role 'maintain' for workspace 'shared-workspace'
 ```
 
+## Data Links
+
+Run `tw data-links -h` to view supported data link operations.  
+
+Data links allow you to work with public and private cloud storage buckets in Data Explorer in the specified workspace. See [Data Explorer][data-explorer] for more information. AWS S3, Azure Blob Storage, and Google Cloud Storage repositories are supported. The full list of operations are:
+
+- `list`: List data links in a workspace
```

Comment:
minor: maybe it's worth to just give a one liner explanation of the two types listed here - cloud data links autodiscovered using workspace credentials (v1-cloud-id) and custom datalinks created by users (v1-user-id)